snap/snapcraft.yaml

```
@@ -223,48 +223,51 @@ parts:
     plugin: dump
     source: https://zoom.us/client/latest/zoom_amd64.deb
     override-build: |
-      # remove danging symlink
-      rm -f $SNAPCRAFT_PART_BUILD/usr/bin/zoom
-      echo "QT_QPA_PLATFORM=xcb \$SNAP/opt/zoom/ZoomLauncher" > $SNAPCRAFT_PART_BUILD/usr/bin/zoom
-      chmod +x $SNAPCRAFT_PART_BUILD/usr/bin/zoom
-      # Unfrig LD_LIBRARY_PATH
-      sed -i s/D_LIBRAR/XXXXXXXX/g opt/zoom/ZoomLauncher
-      snapcraftctl build
+      if [[ `arch` = "x86_64" ]]; then
+        # remove danging symlink
```

```suggestion
        # remove dangling symlink
```

Since you're touching this…